.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # New to Python?
+
+          The issue tracker isn't the right place to get help.
+          Consider instead:
+
+            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+            - posting on [Discourse](https://discuss.python.org/c/users/7)
+            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Bugs in third-party projects (e.g. `requests`) do not belong in the CPython issue tracker
+      options:
+        - label: I am confident this is a bug in CPython, not a bug in a third-party project
+          required: true
+        - label: |
+            I have searched the [CPython issue tracker](https://github.com/python/cpython/issues),
+            and am confident this bug has not been reported before
+          required: true
+  - type: textarea
+    attributes:
+      label: A clear and concise description of the bug
+      description: |
+        Tell us what happened.
+        Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
+        Put any code blocks inside triple backticks.
+
+      value: |
+        ```py
+        your code here
+        ```
```

Comment:
The code fence comes pre-filled now in the response block -- perhaps we could enclose it in comment tags, or put it into the guidance? I think it's a bit disruptive in the response text field itself.
```suggestion
        <!-- 
        ```python
        your code here
        ```
        -->
```

Reply:
So, this was actually a deliberate decision, since we already have the notice about code formatting in the current issue template, but people don't seem to be be paying much attention to that -- I still see a lot of badly formatted code. I thought if we pre-filled the response block, we might have more success there. (Also, trying out my own issue form, I found it quite handy that the backticks were already there for me!)

If we _don't_ want it pre-filled, then the better solution is to do this:

```diff
--- a/.github/ISSUE_TEMPLATE/bug.yml
+++ b/.github/ISSUE_TEMPLATE/bug.yml
@@ -31,7 +31,7 @@ body:
         Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
         Put any code blocks inside triple backticks.

-      value: |
+      placeholder: |
         ```py
         your code here
         ```
```

Reply:
Maybe make it valid syntax, i.e. a comment:

```suggestion
        ```python
        # TODO: Put your code here.
        ```
```

Reply:
```suggestion
        ```python
        # Add your code block here!
        ```
```